packages/component/src/decorator/private/BorderFlair.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { Fragment, memo, useCallback, useState, type ReactNode } from 'react';
 import cx from 'classnames';
+import { useStyles } from 'botframework-webchat-styles/react';
```

Comment:
IntelliSense is not working for me on `classNames.`, is it working for you?

Reply:
Yes, it does:
![Screenshot from 2024-09-27 09-54-54](https://github.com/user-attachments/assets/7ac735e7-1f91-4b47-a31b-a321539968be)

Reply:
![image](https://github.com/user-attachments/assets/db03e61a-a439-4637-9287-ecbf61ab4037)

On my side... don't know why, it's a brand new Codespaces.

Reply:
Note to future travellers: make sure to switch to the workspace version of TypeScript